src/vue/betaflight-logo/BetaflightLogo.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <div id="logo">
```

Comment:
Remove `id="logo"`

Reply:
Currently, we have 2 logo so 2 containers with the same `id="logo"`. So, you must define a class or 2 different id.